src/import/sst_service.rs

```diff
@@ -203,6 +204,20 @@ impl<Router: RaftStoreRouter> ImportSst for ImportSSTService<Router> {
         let timer = Instant::now_coarse();
 
         // Make ingest command.
+        let handle = get_cf_handle(&self.engine, CF_DEFAULT).unwrap();
+        if let Some(n) = get_cf_num_files_at_level(&self.engine, handle, 0) {
```

Comment:
Should we add a flag in IngestRequest to decide whether to enable the flow control? @kennytm

Reply:
@overvenus Perhaps enable it in "normal" mode and disable it in "import" mode?

Reply:
fixed.